docs/_docs/04_reference/02-cli-options.md

```diff
@@ -494,6 +494,11 @@ $ terragrunt stack output --format raw project1_app2.data
 app2
 ```
 
+#### stack clean
+
+Running `terragrunt stack clean` removes the `.terragrunt-stack` directory, which is generated by the `terragrunt stack generate`
+or `terragrunt stack run` commands. This can be useful when you need to reset the stack state or troubleshoot issues.
```

Comment:
```suggestion
or `terragrunt stack run` commands. This can be useful when you need to remove generated configurations or troubleshoot issues.
```

Avoid using the term "state" unless you're talking about OpenTofu/Terraform state.